src/regexmanager.cpp

```diff
@@ -246,18 +286,19 @@ void RegexManager::quote_and_highlight(std::string& str,
 		if (!regex) {
 			continue;
 		}
-		std::string initial_marker = extract_initial_marker(str);
+		std::string outer_marker = "";
```

Comment:
This variable is only used inside the loop, so please move the declaration there.

Reply:
Thank you.